azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -65,8 +65,7 @@
 
 public class ExecutorServlet extends LoginAbstractAzkabanServlet {
 
-  private static final Logger LOGGER =
-      Logger.getLogger(ExecutorServlet.class.getName());
+  private static final Logger logger = Logger.getLogger(ExecutorServlet.class.getName());
```

Comment:
should we change log4j to slf4j?

Reply:
Do you think we should change them for all of the loggers in this PR? Do you know of any risks involved with changes like this?

Reply:
It's more like low hanging fruit given you are modifying this piece already. I can't think of any risk involved considering the usage of the logger in ExecutorServlet.

Reply:
^ cool - will fix :+1: